keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -286,10 +295,13 @@ def __init__(
         anchor_generator=None,
         rpn_head=None,
         rcnn_head=None,
+        prediction_decoder=None,
```

Comment:
can we default `include_rescaling` to None and make it required?  The reason we do this in `keras_cv.models` is to force users to think about how they preprocess their inputs.  Its a huge opportunity for silent failure to assume that they don't want rescaling (or to assume that they do want rescaling!)

Reply:
I know there's been some disagreement about this, but I think for now we should at least make this consistent with the rest of the package, so this sgtm. I'm updating this to make include_rescaling required

Reply:
+1 against a default value here (if that's what you meant).

Reply:
> I know there's been some disagreement about this, but I think for now we should at least make this consistent with the rest of the package, so this sgtm. I'm updating this to make include_rescaling required

If theres disagreement we should discuss.

@fchollet @qlzh727 @martin-gorner and I discussed this thoroughly around a year ago and came to a strong consensus to fight the silent error.  Happy to discuss, but I feel confident in the required API.

Reply:
sgtm. I agree with this decision, I just seem to recall some pushback about it (but it seems like I am likely misremembering!)